py/tests/global_alpha_test.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+
+import pykms
+import time
+
+card = pykms.Card()
+res = pykms.ResourceManager(card)
+#conn = res.reserve_connector("HDMI")
+conn = card.get_first_connected_connector
+crtc = res.reserve_crtc(conn)
+mode = conn.get_default_mode()
+modeb = mode.to_blob(card)
+plane1 = res.reserve_generic_plane(crtc, pykms.PixelFormat.Undefined)
+plane2 = res.reserve_generic_plane(crtc, pykms.PixelFormat.Undefined)
```

Comment:
If you use a specific format, as you seem to do, better give that here. Or if you don't care, just leave the pixelformat argument out.